src/index.css

```diff
@@ -1,5 +1,108 @@
 body {
```

Comment:
you can use rem 
[https://zellwk.com/blog/responsive-vertical-rhythm/](url)
[https://css-tricks.com/rems-ems/](url)